templates/_federalist-redirects.njk

```diff
@@ -203,6 +203,14 @@ server {
   return 301 https://$target_domain;
 }
 
+# modularcontracting.18f.gov to https://github.com/18F/Modular-Contracting-And-Agile-Development
+server {
+  listen {{ PORT }};
+  set $target_domain github.com/18F/Modular-Contracting-And-Agile-Development;
+  server_name modularcontracting.18f.gov;
+  return 301 https://$target_domain;
+}
+
```

Comment:
Looks like we may also need to add a redirect for state-faq to go to GitHub

Reply:
@jmhooper  - wouldn't state-faq redirect to modularcontracting.18f.gov which will redirect github?

Reply:
fyi, @jmhooper agrees there is no need to add a redirect for state-faq to go to GitHub